services/ui-src/src/containers/NewAmendment.js

```diff
@@ -61,6 +61,17 @@ export default function NewAmendment({ fileUpload }) {
       return;
     }
 
+    var validExt = ".png, .gif, .jpeg, .jpg";
```

Comment:
I can't speak to if this works or not.  
Word of caution... in this react code, you're specifying an allowable extentsion list.
In the cloudformation (bucket policy) code, you're specifying a unallowable extension list.  
We should probably do one or the other.

Reply:
Ok. Let me think of a better way to do it.